CHANGELOG.md

```diff
@@ -35,6 +39,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 - Reduces debug log verbosity.
   [cyberark/conjur#2639](https://github.com/cyberark/conjur/pull/2639)
+- Remove status webservices from providers endpoints
```

Comment:
Each changelog entry should go under a single header, not multiple.  I think this makes more sense under `Fixed`.